gfsphysics/GFS_layer/GFS_typedefs.meta

```diff
@@ -2113,9 +2113,9 @@
   units = count
   dimensions = ()
   type = integer
-[aero_in]
-  standard_name = flag_for_aerosol_input_MG
-  long_name = flag for using aerosols in Morrison-Gettelman MP
+[iaerclm]
+  standard_name = flag_for_aerosol_input_MG_radiation
+  long_name = flag for using aerosols in Morrison-Gettelman MP_radiation
```

Comment:
Why is climatological aerosol specific to MG microphysics (in radiation)?

Reply:
@SMoorthi-emc Other microphysics are using different aerosol climatology input. For instance, Thompson MP uses a completely different input dataset of water-friendly and ice-friendly 3d aerosol distributions. As unfortunate as this is, it seems to me that these days aerosols are very much microphysics dependent. Hopefully the developers can at some point agree to use a common data set and format.